src/site/content/en/react/how-amp-can-guarantee-fastness-in-your-nextjs-app/index.md

```diff
@@ -0,0 +1,99 @@
+---
+layout: post
+title: How AMP can guarantee fastness in your Next.js app
+authors:
+  - houssein
+subhead: |
+  Learn about the benefits and tradeoffs of supporting AMP in your Next.js app
```

Comment:
FYI I like this quite a bit. It reflects the general direction that we want to take web.dev.